packages/headless/src/controllers/commerce/search/did-you-mean/headless-did-you-mean.ts

```diff
@@ -37,8 +36,6 @@ export function buildDidYouMean(engine: CommerceEngine): DidYouMean {
   const controller = buildController(engine);
   const getState = () => engine[stateKey].didYouMean;
 
-  const hasQueryCorrection = () => hasQueryCorrectionSelector(getState());
```

Comment:
This selector returns `state.queryCorrection.correctedQuery !== '' || state.wasCorrectedTo !== ''`, however if `correctedQuery === '', `wasCorrectedTo` will always be `''` so logic is superflous.